index.d.ts

```diff
@@ -0,0 +1,78 @@
+export enum ZigbeeDeviceAccess {
+    STATE = 1,
+    SET = 2,
+    STATE_SET = 3,
+    STATE_GET = 5,
+    ALL = 7,
+}
+
+export interface ZigbeeDeviceExposeGeneric {
+    access: ZigbeeDeviceAccess
+    description: string
+    name: string
+    property: string
+    type: string
+}
+
+export interface ZigbeeDeviceExposeEnum {
```

Comment:
I have my own implementation, you might want to have a look
https://github.com/nurikk/z2m-frontend/blob/a341b4dd5704d645c11845deb2213915119a0e03/src/types.ts#L100

Reply:
I thought about extending also. Makes it easier to grab the correct type from the definitions. 👍